conjur-api/Client.Methods.cs

```diff
@@ -36,6 +36,28 @@ public IEnumerable<Variable> ListVariables(string query = null)
             return Conjur.Variable.List(this, query);
         }
 
+        /// <summary>
+        /// Create an onject representing the User with this specified name.
+        /// </summary>
+        /// <returns>The user.</returns>
+        /// <param name="name">Name.</param>
+        /// <returns>User object.</returns>
+        /// <seealso cref="User()"/>
```

Comment:
Please pay closer attention to docstrings. This one has two `<returns>`. Also they should be more verbose than just repeating the method signature.